governance/api.go

```diff
@@ -226,8 +226,8 @@ func (api *GovernanceAPI) GetRewardsAccumulated(first rpc.BlockNumber, last rpc.
 	accumRewards.TotalBurntTxFee = blockRewards.BurntFee
 	accumRewards.TotalProposerRewards = blockRewards.Proposer
 	accumRewards.TotalStakingRewards = blockRewards.Stakers
-	accumRewards.TotalKFFRewards = blockRewards.KFF
-	accumRewards.TotalKCFRewards = blockRewards.KCF
+	accumRewards.TotalKFFRewards = blockRewards.KIF
```

Comment:
Don't we also need to update `AccumulatedRewards`?

Reply:
Right, fixed ab09c4c4